internal/mode/static/nginx/config/policies/processor.go

```diff
@@ -0,0 +1,27 @@
+package policies
+
+// UpstreamSettingsProcessor is an interface that defines a method for processing a list of policies
+// into upstream settings.
+type UpstreamSettingsProcessor interface {
+	Process(policies []Policy) UpstreamSettings
+}
+
+// UpstreamSettings contains settings from UpstreamSettingsPolicy.
+type UpstreamSettings struct {
+	// ZoneSize is the zone size setting.
+	ZoneSize string
+	// KeepAlive contains the keepalive settings.
+	KeepAlive KeepAlive
```

Comment:
Could we not just reuse the UpstreamKeepAlive struct defined in config.go? That's ultimately what we're converting this into, right?

Reply:
I see a benefit of decreasing our usage of shared types between packages. I think that might be why Kate suggested to keep separate types.

Reply:
I like having the processor in the `/policies/upstreamsettings`. If we can keep that structure and reuse the type, I'm good with that.

Reply:
I think I did this correctly.